packages/catalog-process/test/catalogService.integration.test.ts

```diff
@@ -1639,139 +1724,248 @@ describe("database test", async () => {
     });
 
     describe("clone descriptor", () => {
-      beforeAll(() => {
-        vi.useFakeTimers();
-        vi.setSystemTime(new Date("2024-01-01T12:00:00"));
-      });
-      afterAll(() => {
-        vi.useRealTimers();
-      });
```

Comment:
If you add this

```
beforeAll(() => {
    vi.useFakeTimers();
    vi.setSystemTime(new Date());
});
afterAll(() => {
    vi.useRealTimers();
 });
```
you can remove the `{ retry: 3 }` below.
In this way we ensure that every call to `new Date()` both in the logic and in the test will result to the same instant.

Reply:
True, thanks for the suggestion! Done